BACKLOG.md

```
@@ -6,18 +6,15 @@
 
 ---
 
-### [AI] Migrate provider to OpenRouter or Vercel AI SDK
-- support arbitrary set of model compositions for question generation and other ai / llm / generative features
-- sometimes we want gemini-2.5-flash, sometimes we want gpt-5-mini, sometimes we want something else etc
+### [INFRA] Optimize ConvexDB database bandwidth -- or Migrate off ConvexDB
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)